extensions/gmsa-dc/v1/Promote-DC.ps1

```diff
@@ -0,0 +1,51 @@
+###################################################################################
+# This script will automate the creation of a Forest, Domain, and Domain Controller
+#
+# This is only for automated e2e testing.  DO NOT use this for production.
+# Jeremy Wood (JeremyWx)
+# Version: 1.0.0.0
+###################################################################################
+
+# Create a working directory and change to it
+mkdir C:\gmsa
+Set-Location -Path C:\gmsa
+
+# Uncomment line below for troubleshooting

```

Comment:
Let's remove this, or actually comment the line below?

Reply:
I changed the comment to:  # Logging for troubleshooting

Will that work?